stetho-okhttp/src/test/java/com/facebook/stetho/okhttp/StethoInterceptorTest.java

```diff
@@ -50,24 +60,37 @@
   @Rule
   public PowerMockRule rule = new PowerMockRule();
 
-  @Test
-  public void testHappyPath() throws IOException {
+  private NetworkEventReporter mMockEventReporter;
+  private StethoInterceptor mInterceptor;
+  private OkHttpClient mClientWithInterceptor;
+
+  @Before
+  public void setUp() {
     PowerMockito.mockStatic(NetworkEventReporterImpl.class);
 
-    final NetworkEventReporter mockEventReporter = Mockito.mock(NetworkEventReporter.class);
-    InOrder inOrder = Mockito.inOrder(mockEventReporter);
-    Mockito.when(mockEventReporter.isEnabled()).thenReturn(true);
-    ByteArrayOutputStream capturedOutput = hookAlmostRealInterpretResponseStream(mockEventReporter);
-    PowerMockito.when(NetworkEventReporterImpl.get()).thenReturn(mockEventReporter);
+    mMockEventReporter = Mockito.mock(NetworkEventReporter.class);
+    Mockito.when(mMockEventReporter.isEnabled()).thenReturn(true);
+    PowerMockito.when(NetworkEventReporterImpl.get()).thenReturn(mMockEventReporter);
 
-    StethoInterceptor interceptor = new StethoInterceptor();
+    mInterceptor = new StethoInterceptor();
+    mClientWithInterceptor = new OkHttpClient();
+    mClientWithInterceptor.networkInterceptors().add(mInterceptor);
+  }
+
+  @Test
+  public void testHappyPath() throws IOException {
```

Comment:
Ack'd, but it should be `testManualInterceptorNoCompression` -- there is already a `testWithResponseCompression`...